ci_framework/roles/devscripts/tasks/sub_tasks/31_repo.yml

```diff
@@ -0,0 +1,72 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Clone the dev-scripts repository.
+  ansible.builtin.git:
+    repo: "{{ cifmw_devscripts_repo }}"
+    dest: "{{ cifmw_devscripts_repo_dir }}"
+    depth: 1
+    single_branch: true
+    force: true
+    version: HEAD  # noqa: latest[git]
+  register: clone_out
+  retries: 3
+  delay: 15
+  until: "clone_out is not failed"
+
+# This is the same patch as proposed upstream with
+# https://github.com/openshift-metal3/dev-scripts/pull/1595
+# Even if it gets merged, the patch here won't fail.
+- name: Patch dev-scripts for libvirt stability
```

Comment:
we *may* not need it if we "hack" the virtproxyd.service Service to remove the timeout, like done in #720 . I didn't test without restarting the service, but I think it should do the trick